Code/EnvironmentMetricsService/src/main/java/ie/tcd/wayfinder/envmetrics/init/Oauth2ClientConfig.java

```diff
@@ -0,0 +1,17 @@
+package ie.tcd.wayfinder.envmetrics.init;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.oauth2.client.OAuth2ClientContext;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.security.oauth2.client.resource.OAuth2ProtectedResourceDetails;
+import org.springframework.security.oauth2.config.annotation.web.configuration.EnableOAuth2Client;
+
+//@Configuration
+//@EnableOAuth2Client
+public class Oauth2ClientConfig {
+//	@Bean
+//	public OAuth2RestTemplate oauth2RestTemplate(OAuth2ClientContext oauth2ClientContext, OAuth2ProtectedResourceDetails details) {
```

Comment:
is this meant to be commented??

Reply:
This was done just to turn off Auth. Will uncomment it